app/templates/photo-albums/photo-album/photos/index.hbs

```diff
@@ -1,7 +1,7 @@
 {{#ember-wormhole to="navbar-wormhole"}}
   {{#page-actions-buttons pageActions}}
     <div class="col col-md-auto page-action">
-      <button class="btn btn-primary" {{action "downloadAlbum"}}>
+      <button class="btn btn-primary" {{action "openModal"}}>
```

Comment:
Dit kam waarschijnlijk wel met een `mut` helper oid

Reply:
Dat is nieuw voor mij, wat is een `mut` helper?

Reply:
Ik denk dat deze gist het beter kan uitleggen dan ik zelf:
https://gist.github.com/Joelkang/dd7f3f54dc2f1115526dbca001853dbe

Reply:
`{{action (mut modalIsOpen) value=true}}`

Reply:
Ah, zoek ik uit.